patroni/config.py

```diff
@@ -88,6 +89,10 @@ def config_file(self):
     def dynamic_configuration(self):
         return deepcopy(self._dynamic_configuration)
 
+    @property
+    def is_standby_cluster(self):
```

Comment:
There is no need to have `is_standby_cluster` property here.

The only reason why we keep information about `standby_cluster` in the `Config()` is just to be able to dump into `patroni.dynamic.json`. It makes it possible to restore it if somebody accidentally wipes DCS out.

Reply:
There is such need - because we need to get this info directly from config file before bootstrap. It's not really good to read it directly from `_dynamic_configuration`, so I've created a property.